tests/WeightListTest.php

```diff
@@ -45,8 +47,22 @@ public function testGetValue()
         $this->assertEquals('lemon', $method->invoke($this->weightList, 71));
 
         $this->assertEquals('potato', $method->invoke($this->weightList, 100));
+
     }
-    
+
+    /**
+     * @expectedException \OutOfBoundsException
+     */
+    public function testGetValueWithEmptyWeightList()
```

Comment:
private method not required to be tested because private method seen only in class and may be removed or replaced. Unit tests must test only specification of class and don't know nothing about internal implementation of class. Also usage of reflection must be minimized, because it is not obviously and broce IDE's refactoring mechanisms.